.goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    #- go mod download
+    # you may remove this if you don't need go generate
+    #- go generate ./...
+    - make generate
+    - git status
+builds:
```

Comment:
No docker image option?

Reply:
We already have docker push in another action